indico/modules/core/controllers.py

```diff
@@ -269,3 +270,25 @@ def _serialize_principal(self, identifier, principal):
     def _process(self, values):
         return jsonify({identifier: self._serialize_principal(identifier, principal)
                         for identifier, principal in values.viewitems()})
+
+
+class RHSignURL(RHProtected):
+    def _process(self):
+        endpoint = request.json.get('endpoint')
+        if not endpoint:
+            raise BadRequest
+        # filter out non-standard args
+        url_params = request.json.get('url_params', {})
+        url_params = {k: v for k, v in url_params.viewitems() if not k.startswith('_')}
+        query_params = request.json.get('query_params', {})
+        query_params = {k: v for k, v in query_params.viewitems() if not k.startswith('_')}
+        url = signed_url_for(session.user, endpoint, url_params=url_params, _external=True, **query_params)
+        Logger.get('url_signing').info("Created signed URL for endpoint '%s' (%s)", endpoint, url)
```

Comment:
maybe including the signature in the logged url isn't such a great idea.. logging the user who created it is probably more useful

Reply:
I thought of that too, but is it much different from what we get in webserver logs?

Reply:
good point ;)